src/utils.ts

```diff
@@ -74,3 +74,27 @@ export default function formatHumanDate(date: Date) {
     day: "numeric",
   });
 }
+
+export interface ImageData {
+  width: number;
+  height: number;
+}
+
+export function scaleImage(
+  image: ImageData,
+  maxWidth: number,
+  maxRatio: number,
+): ImageData {
+  if (image.width > maxWidth) {
+    const ratio =
+      image.width / image.height < maxRatio
+        ? maxRatio
+        : image.width / image.height;
+    const width = maxWidth;
+    const height = Math.round(width / ratio);
+
+    return { width, height };
+  } else {
+    return image;
+  }
+}
```

Comment:
I think if we just take the image we get from data outside and use the data object type as input, we just use the image.height anyway